tests/GrabyFunctionalTest.php

```diff
@@ -583,4 +583,25 @@ public function testJsonLd()
         $this->assertCount(1, $res['authors']);
         $this->assertSame('Jeremy Goujon', $res['authors'][0]);
     }
+
+    public function testEmptyNodesRemoved()
+    {
+        $response = new Response(
+            200,
+            [
+                'content-type' => 'text/html',
+            ],
+            Stream::factory(file_get_contents(__DIR__ . '/fixtures/sites/framablog.test'))
+        );
+        $client = new Client();
+        $client->getEmitter()->attach(new Mock([$response]));
```

Comment:
You can remove the mock, it is not used and it breaks the build: https://travis-ci.org/j0k3r/graby/jobs/388131182#L490